src/main/java/org/json/XMLParserConfiguration.java

```diff
@@ -61,16 +61,19 @@ public class XMLParserConfiguration extends ParserConfiguration {
      */
     private Set<String> forceList;
 
+    private boolean shouldTrimWhiteSpace;
```

Comment:
Please include a comment block that describes the purpose of this flag, and all of the ways it changes the behavior of the library.

Reply:
Will update